modules/javafx.graphics/src/main/java/javafx/scene/layout/Background.java

```diff
@@ -353,6 +353,19 @@ public Background(final @NamedArg("fills") BackgroundFill[] fills, final @NamedA
         hash = result;
     }
 
+    /**
+     * A convenience factory method for creating a background with a single {@code Paint}. The background will use the
+     * default {@code CornerRadii} and the default {@code Insets} of {@link BackgroundFill}.
+     *
+     * @implSpec This call is equivalent to {@code new Background(new BackgroundFill(fill, null, null));}.
+     * @param fill the fill of the background
+     * @return a new background of the given fill
+     * @since 18
+     */
+    public static Background fill(Paint fill) {
+        return new Background(new BackgroundFill(fill, null, null));
```

Comment:
null **CornerRaddii** and null **Insets** will use **CornerRadii.EMPTY** and **Insets.EMPTY**. Maybe we should use those here instead so it's more clear for anyone having a look in the source code? I also always use those instead of null.
Same for **BorderStroke**

Reply:
Are you talking about the implementation or the code? I guess both, since the `@implSepc` indicates what this call is equivalent to. I don't have a strong opinion on this one.

Reply:
You don't need to look at the source code, I can link to the delegated constructor and the docs will show those. I prefer using `null`s because the point of these is that you want a border/background with 1 color and you don't care about things like widths, insets and corners - they default to whatever the default is. If you care what they are, this method is probably not what you're looking for. It's also shorter and less to read through when the extra info is defaults.